### PR TITLE
Fix failure during serving installation in e2e test

### PR DESCRIPTION
**Pull request by zhongduo**
Install serving in case of failure to avoid race condition of webhook and config map.

### CODE REVIEW

File: test/lib.sh [hunk 1]
```diff
@@ -40,7 +40,8 @@ function start_knative_gcp_monitoring() {
 
 # Install all required components for running knative-gcp.
 function start_knative_gcp() {
-  start_latest_knative_serving || return 1
+  # Try reapply serving yaml again in case of race condition between webhook and config map
+  start_latest_knative_serving || start_latest_knative_serving || return 1
```
Comment (Harwayne) [inline test/lib.sh:44]: Wow, we are really desperate.

**Review by Harwayne — COMMENTED** [review]
/lgtm

File: test/lib.sh (hunk 1 above)
Comment (zhongduo) [reply test/lib.sh:44]: We are doing that for eventing too before the webhook is ignored.

**Comment by zhongduo** [comment]
@Harwayne some jobs hanged in the previous one, I just pushed it again without change.

**Review by Harwayne — COMMENTED** [review]
/lgtm

**Pull request merged**